projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,582 @@
+	"block.projectexpansion.emc_link.empty_hand": "清空你的手来取回物品。",
+	"block.projectexpansion.emc_link.cleared": "输出物品已被清除。",
+	"block.projectexpansion.emc_link.not_enough_emc": "你没有足够的EMC购买这个，你需要%s。",
+	"block.projectexpansion.emc_link.set": "输出物品已设置为%s。",
+	"block.projectexpansion.emc_link.no_emc_value": "%s没有EMC。",
+	"block.projectexpansion.emc_link.no_export_remaining": "物品输出限制已经达到，请稍等。",
+	"block.projectexpansion.basic_emc_link": "基本EMC接口1型",
+	"block.projectexpansion.dark_emc_link": "暗物质EMC接口2型",
+	"block.projectexpansion.red_emc_link": "红物质EMC接口3型",
+	"block.projectexpansion.magenta_emc_link": "品红物质EMC接口4型",
+	"block.projectexpansion.pink_emc_link": "粉物质EMC接口5型",
+	"block.projectexpansion.purple_emc_link": "紫物质EMC接口6型",
+	"block.projectexpansion.violet_emc_link": "紫罗兰物质EMC接口7型",
+	"block.projectexpansion.blue_emc_link": "蓝物质EMC接口8型",
+	"block.projectexpansion.cyan_emc_link": "青物质EMC接口9型",
+	"block.projectexpansion.green_emc_link": "绿物质EMC接口10型",
+	"block.projectexpansion.lime_emc_link": "黄绿物质EMC接口11型",
+	"block.projectexpansion.yellow_emc_link": "黄物质EMC接口12型",
+	"block.projectexpansion.orange_emc_link": "橙物质EMC接口13型",
+	"block.projectexpansion.white_emc_link": "白物质EMC接口14型",
+	"block.projectexpansion.fading_emc_link": "渐消物质EMC接口15型",
+	"block.projectexpansion.final_emc_link": "终极EMC接口16型",
+	"block.projectexpansion.magenta_fuel_block": "品红燃料块",
+	"block.projectexpansion.pink_fuel_block": "粉燃料块",
+	"block.projectexpansion.purple_fuel_block": "紫燃料块",
+	"block.projectexpansion.violet_fuel_block": "紫罗兰燃料块",
+	"block.projectexpansion.blue_fuel_block": "蓝燃料块",
+	"block.projectexpansion.cyan_fuel_block": "青燃料块",
+	"block.projectexpansion.green_fuel_block": "绿燃料块",
+	"block.projectexpansion.lime_fuel_block": "黄绿燃料块",
+	"block.projectexpansion.yellow_fuel_block": "黄燃料块",
+	"block.projectexpansion.orange_fuel_block": "橙燃料块",
+	"block.projectexpansion.white_fuel_block": "白燃料块",
+	"block.projectexpansion.magenta_matter_block": "品红物质块",
+	"block.projectexpansion.pink_matter_block": "粉物质块",
+	"block.projectexpansion.purple_matter_block": "紫物质块",
+	"block.projectexpansion.violet_matter_block": "紫罗兰物质块",
+	"block.projectexpansion.blue_matter_block": "蓝物质块",
+	"block.projectexpansion.cyan_matter_block": "青物质块",
+	"block.projectexpansion.green_matter_block": "绿物质块",
+	"block.projectexpansion.lime_matter_block": "黄绿物质物质块",
+	"block.projectexpansion.yellow_matter_block": "黄物质块",
+	"block.projectexpansion.orange_matter_block": "橙物质块",
+	"block.projectexpansion.white_matter_block": "白物质块",
+	"block.projectexpansion.fading_matter_block": "渐消物质块",
+	"block.projectexpansion.arcane_table": "炼金术桌",
+	"block.projectexpansion.transmutation_interface": "转化接口",
+	"block.projectexpansion.transmutation_interface.tooltip": "可以与兼容的模组如Applied Energistics一起使用，以访问你已学习过的物品。",
+	"block.projectexpansion.compact_sun": "压缩太阳",
+	"block.projectexpansion.compact_sun.tooltip": "太阳之力尽在其中。",
+	"block.projectexpansion.power_flower.tooltip": "每%stick生产一次EMC%s。（20/秒）",
```

```suggestion
	"block.projectexpansion.power_flower.tooltip": "每%s刻生产一次EMC%s。（20/秒）",
```
tick建议翻译出来
游戏里的显示情况？好奇为啥会有俩插入字符串